di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/CoreStub.java

```diff
@@ -72,23 +62,10 @@ private RSAKey getSigningKeystore() throws Exception {
                 RSAKey.load(keystore, CoreStubConfig.CORE_STUB_KEYSTORE_ALIAS, keyStorePassword));
     }
 
-    private ECKey getEcPrivateKey()
-            throws NoSuchAlgorithmException, InvalidKeySpecException, ParseException {
-        EncodedKeySpec privateKeySpec =
-                new PKCS8EncodedKeySpec(
-                        Base64.getDecoder().decode(CoreStubConfig.CORE_STUB_SIGNING_PRIVATE_KEY));
-        return new ECKey.Builder(ECKey.parse(CoreStubConfig.CORE_STUB_SIGNING_PUBLIC_JWK))
-                .privateKey(
-                        (ECPrivateKey) KeyFactory.getInstance("EC").generatePrivate(privateKeySpec))
-                .build();
-    }
-
-    private RSAPublicKey getEncryptionPublicKey()
-            throws NoSuchAlgorithmException, InvalidKeySpecException {
-        byte[] encodedKeySpec =
-                Base64.getDecoder().decode(CoreStubConfig.CORE_STUB_ENCRYPTION_PUBLIC_KEY);
-        KeyFactory keyFactory = KeyFactory.getInstance("RSA");
-        X509EncodedKeySpec keySpec = new X509EncodedKeySpec(encodedKeySpec);
-        return (RSAPublicKey) keyFactory.generatePublic(keySpec);
+    private ECKey getEcPrivateKey() throws ParseException {
+        return ECKey.parse(
+                new String(
+                        Base64.getDecoder()
+                                .decode(CoreStubConfig.CORE_STUB_SIGNING_PRIVATE_KEY_JWK_BASE64)));
```

Comment:
This is a simpler way to create an EC key from the base64 encoded public key JWK set in the config repo